components/filter/filter.js

```diff
@@ -256,7 +258,10 @@ class Filter extends LocalizeCoreElement(RtlMixin(LitElement)) {
 		let dimensionHTML;
 		switch (dimension.type) {
 			case 'd2l-filter-dimension-set':
-				dimensionHTML = html`<div aria-live="polite">${this._createSetDimension(dimension)}</div>`;
+				dimensionHTML = html`
+				<div aria-live="polite" id="${ifDefined(singleDimension ? this._filterContainerId : undefined)}">
```

Comment:
Only adding this id if the filter has a single dimension, otherwise it will be duplicated causing axe errors.

Reply:
Can we use a class instead?  And if any new dimensions are added that have the same problem, we can put the class on them as well.